doc/md/timers.md

```diff
@@ -0,0 +1,26 @@
+# Timers
+
+Internet Computer canisters can set an arbitrary number of single-expiration or recurring timers. See the `Timers.mo` module in the base library.
+
+A simple, contrived example is a periodic reminder, that logs a new-year's message:
+
+``` motoko file=./examples/Reminder.mo
```

Comment:
```suggestion
``` motoko no-repl file=./examples/Reminder.mo
```

Reply:
Could you add a no-repl to the hearbeat example too, while you've got it checked out?